text2sql-backend/services.py

```diff
@@ -22,7 +22,7 @@ class SQLResults(TypedDict):
 
 class SchemaService:
     @classmethod
-    def extract_tables(cls, conn: Connection, connection_id: str) -> dict[str, dict[str, TableField]]:
+    def extract_tables(cls, conn: SQLiteConnection, connection_id: str) -> dict[str, dict[str, TableField]]:
```

Comment:
This shouldn't be specific to sqlite. Should be generic supporting any sqlalchemy supported DB. Any reason why you wanted SQLITE typing?

Reply:
Oh I see I have that in some places too..; ignore then we'll fix it later

Reply:
Heheh yeah was just fixing annoying type errors